assets/script.js

```diff
@@ -0,0 +1,203 @@
+// Declaring global variables
+var today = dayjs();
+var full_name = "";
+var index;
+var tarotDesc = [];
+var tarotName = [];
+var historySign;
+var historyName;
+var tarotID;
+
+today = today.format('MM/DD/YYYY');
+// Empty the current horoscope section of the page
+function clear() {
+    console.log("inside clear");
+    $(".curr-hor").empty();
+}
+// function to display tarto information with a modal for the selected card
+function updatetarot(tarotData) {
+    console.log("inside     update tarot", tarotData);
```

Comment:
I'm going to stop repeating the comment for `console.log` just try to remove these when you submit code for production.